srcs/utils/env_list_get_set.c

```diff
@@ -44,9 +51,13 @@ void	set_env_value(char *name, char *value)
 {
 	t_list	*to_set;
 
-	to_set = get_env_lst(name);
+	if (!value)
+		return ;
+	to_set = get_env_lst(name, NULL);
 	if (!to_set)
 		return ;
-	free(((t_env *)to_set->content)->value);
+	if (((t_env *)to_set->content)->value)
+		free(((t_env *)to_set->content)->value);
+	((t_env *)to_set->content)->value = NULL;
```

Comment:
Hum, here don't you need to do that only if you free it ?

Reply:
setting it to NULL you mean? well, if I don't need to free it, it's because it's NULL, so I'm setting NULL to NULL

Reply:
Yeah exactly. But well, I guess redundancy doesn't hurt.
Let's merge this.